packages/kbn-search-api-panels/components/ingest_pipelines/ingest_pipeline_panel.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState, useMemo } from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiBadge,
+  EuiSuperSelect,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { IngestGetPipelineResponse } from '@elastic/elasticsearch/lib/api/types';
+import { createIngestPipelineOptions } from './ingest_pipeline_options';
+
+interface IngestPipelinePanelProps {
+  setSelectedPipeline: (pipeline: string) => void;
```

Comment:
See other comment but we should add `selectedPipeline` as a prop as well.

Reply:
Updated